setup.cfg

```diff
@@ -45,7 +45,7 @@ install_requires =
 
 [options.extras_require]
 channels =
-    Django>=2.2, <2.3
-    channels==1.1.8
-    asgi_redis
-    graphql_ws
+    Django>=2.2
+    channels==3.0.4
```

Comment:
https://github.com/GrappleGQL/wagtail-grapple/pull/184/files#diff-e76e3d4b8b9857a8082a63501a2453c42fc867bd2900f1cc814c2b87c4feb9ccR5 says `channels==2.4.0` which is different from this 🤔